### DISCUSSION TITLE
Unable to submit your application. Please try again

### DISCUSSION THREAD

**Discussion by yunusmertofficial** [community/community#70667] [GitHub Education] [answered]
### Hi everyone,

I am a student and I have a student email address. Even though I did everything right, I can't get the github student package. I get this error when I try to process my student document after sending it.

**Comment by antonio-jotta** [comment]
Try a new browser. I was on OperaGX and it wasn't working, but switching to Edge solved the issue

**Comment by Gyanendu01** [comment]
Hey yunusmertofficial,

Ensure that you meet all the eligibility criteria for the GitHub Student Developer Pack. This includes being a student at an accredited institution and having a valid student email address. Review the eligibility requirements on GitHub's Education page to make sure you qualify.

 Double-check the document you submitted for verification. Make sure that the document is clear, up-to-date, and shows your student status. Commonly accepted documents include student IDs, enrollment letters, or transcripts.


Try Using a new browser, I was also facing the same issue which was resolved by following considerations.

**Comment by yunusmertofficial** [comment]
thank you very much, When I downloaded it in another browser, I got this error "Your application cannot be reviewed until you fix the following:
Please secure your GitHub account with two-factor authentication.
The GitHub account you are using was recently created, and you appear to be having difficulty demonstrating your academic affiliation. Please wait a few days then try again."

**Reply by Cayvanveen** [reply, marked as answer]
Yeah it makes it suspicious if you do it all on the same day. And i recommend turning 2 step verification on. You can do it with sms too, which is the most simple

**Reply by PhiHoang12032003** [reply]
help me with 2 step verification

**Comment by yunusmertofficial** [comment]
Thank you very much, the information you gave me was very useful.

**Comment by anzenchitai** [comment]
Last year, I was able to activate the pack on my account using my academic e-mail address and student ID. Now it doesn't let me. So far I've also tried using an acceptance letter and a transcript. I'm done using Github.

**Comment by gillamir86** [comment]
when i submit my student id it shows that i have already used it, can you help me to submit and avail global campus benefits

**Reply by samus-aran** [reply]
Hi @gillamir86 
Try taking another photo of the Student ID. We do prefer live photos captured during the application. Hope this helps 👍

**Comment by sarahbstein** [comment]
I turned 2FA on and tried to resubmit my application, but it keeps saying that I already used the proof I uploaded. I'm so frustrated because I don't have any other type of proof (this is an official letter from the university). When I upload something else, it just reloads the page and doesn't submit. I haven't been able to get in contact with GitHub Support. Please help!
